src/main/java/org/jitsi/videobridge/IceUdpTransportManager.java

```diff
@@ -1445,7 +1445,7 @@ public BundleContext getBundleContext()
      * {@inheritDoc}
      */
     @Override
-    public DtlsControlImpl getDtlsControl(Channel channel)
+    public DtlsControlImpl getSrtpControl(Channel channel)
```

Comment:
The base method (in TransportManager) is called getSrtpControl and returns SrtpControl. IceUdpTransportManager uses DTLS specifically (because it has to set fingerprints for example), and as such return the more specific class. The caller can use SrtpControl if it doesn't care what the implementation is, or if it knows that the transport manager is IceUdpTransportManager, it can treat the value as DtlsControlImpl (without a cast). I don't think this defeats the purpose of the hierarchy.